src/main/kotlin/calculator/domain/Operator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator.domain
+
+import java.lang.IllegalArgumentException
+
+enum class Operator(
+    private val value: String,
+    val operation: (Operand, Operand) -> Operand
+) {
+    PLUS("+", { num1, num2 -> Operand(num1.number + num2.number) }),
+    MINUS("-", { num1, num2 -> Operand(num1.number - num2.number) }),
+    MULTIPLE("*", { num1, num2 -> Operand(num1.number * num2.number) }),
+    DIVIDE(
+        "/",
+        { num1, num2 ->
+            if (num2.number == 0.0)
+                throw IllegalArgumentException("0으로는 나눌 수 없습니다")
+            Operand(num1.number * num2.number)
```

Comment:
```suggestion
            Operand(num1.number / num2.number)
```
이렇게 되어야 할 것 같네요 :)
각각의 연산자들에 대해 테스트도 추가해주는게 어떨까요?